### ISSUE TITLE
[Bug Report][3.1.2] Menu dropdown item jumps away after resize the page

### ISSUE THREAD

**Issue by velly** [vuetifyjs/vuetify#16469]
### Environment
**Vuetify Version:** 3.1.2
**Vue Version:** 3.2.45
**Browsers:** Chrome 108.0.0.0
**OS:** Windows 10

### Steps to reproduce
0: hover on v-btn => menu item shows under the trigger button.
1: resize the page
2: hover on v-btn => menu item shows but jumps away from trigger button

### Expected Behavior
menu dropdown items should always rendered under the trigger button.

### Actual Behavior
menu dropdown items  jumps away.

### Reproduction Link
[https://codesandbox.io/s/vue-3-composition-api-script-setup-example-forked-91dy3x?file=/src/components/LocaleSwitcher.vue:301-323](https://codesandbox.io/s/vue-3-composition-api-script-setup-example-forked-91dy3x?file=/src/components/LocaleSwitcher.vue:301-323)

**Comment by velly** [comment]
@KaelWD  Hi Kael, I reproduced the issue and create a new issue for dropdown menu jumping away.

**Comment by davidzwa** [comment]
I experience the same in my Vuetify 3.1.2 project by opening and closing the menu once, after opening it a second time the menu covers the activator button.

Some helpful information:
1) (1st opening) The menu can exceed page height (adding a bottom margin fixes this for me)
2) (2nd opening of menu) The button is covered, while it should not be, but the page height is now not exceeded (but the added bottom margin, from **case 1** now causes a unnecessary gap)

**Comment by davidzwa** [comment]
I can confirm that it can be patched (concerning triggering the activator) by checking the amounts of time the menu was opened.

This was my fix added to the first child of `<v-menu> </v-menu>` which is after the `<template></template>` with activator button.
```
  :style="openings <= 1 ? 'margin-bottom: 40px' : 'margin-top: 40px'"
```

**Comment by ColiZei** [comment]
I can confirm this.

I use `v-menu` in a `v-app-bar`. After opening the drawer, the opening position of the dropdown is wrong.

**Comment by KaelWD** [comment]
Duplicate of #15352

**Comment by KaelWD** [comment]
@davidzwa That's unrelated, might have been fixed by a11138e5

**Comment by davidzwa** [comment]
Ok, I just checked but it is not fixed with 3.1.3 and https://github.com/vuetifyjs/vuetify/commit/a11138e5f75a9ef81bb3b39daeb8bd1cb07997cb
